p256/src/ecdsa/recoverable.rs

```diff
@@ -0,0 +1,382 @@
+//! Ethereum-style "recoverable signatures".
```

Comment:
This is an Ethereum-specific signature format, and one which I would not encourage using in other applications.